packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -479,10 +479,9 @@ abstract class Widget extends DiagnosticableTree {
 /// ```
 /// {@end-tool}
 ///
-/// By convention, widget constructors only use named arguments. Named arguments
-/// can be marked as required using [@required]. Also by convention, the first
-/// argument is [key], and the last argument is `child`, `children`, or the
-/// equivalent.
+/// By convention, widget constructors only use named arguments. Also by
+/// convention, the first argument is [key], and the last argument is `child`,
+/// `children`, or the equivalent.
```

Comment:
do we not want to mention `required`?

Reply:
I feel that as required is now a Dart keyword and the linter will complain if a non-nullable parameter is not `required`, the entire sentence is unnecessary.

If, however, you feel that the sentence is to be kept, I can revert this file but only removing the `@` sign.

Reply:
I don't have a strong opinion one way or the other.

Reply:
Maybe we could get other opinions.

Reply:
@xster @jonahwilliams @abarth What do you think?

Reply:
seems reasonable to leave that to a more generic language tour

Reply:
May I solve it, then?

Reply:
you don't need to click the resolve button, but otherwise I think yes we've resolved this :)